test/tests/outputs/format/test_json.py

```diff
@@ -11,8 +11,8 @@ def find_app():
     """
     moose_dir = os.environ.get("MOOSE_DIR")
     if not moose_dir:
-        file_dir = os.dirname(os.path.abspath(__file__))
-        moose_dir = os.dirname(os.dirname(file_dir))
+        file_dir = os.path.dirname(os.path.abspath(__file__))
+        moose_dir = os.path.abspath(os.path.join(file_dir, "..", "..", "..", ".."))
```

Comment:
Oh, I see what you mean, just use those lines to get the top level directory. Will do.